tox.ini

```diff
@@ -25,7 +25,7 @@ whitelist_externals =
 commands_pre =
     py{36,37,38}: pip install fastparquet
 commands =
-    py{35,36,37,38}: ipython kernel install --user --name=wandb_python
+    py{35,36,37,38,39}: ipython kernel install --user --name=wandb_python
```

Comment:
@raubitsj not sure why this is here, did we remove 39 from master for a reason?

Reply:
No, i never added it to this ipython section.  
py39 in master only runs a limited amount of tests because of package incompatibilities.
If you have something that works in py39, please add it (and feel free to remove any skips that are no longer needed)
torch didnt support py39 last i checked.